TestFoundation/TestFTP.swift

```diff
@@ -0,0 +1,116 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 20188888888 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+#if DEPLOYMENT_RUNTIME_OBJC || os(Linux)
+    import Foundation
+    import XCTest
+#else
+    import SwiftFoundation
+    import SwiftXCTest
+#endif
+
+class TestFTP : LoopbackFTPServerTest {
+    
+    static var allTests: [(String, (TestFTP) -> () throws -> Void)] {
+        return [
+            
+            ("test_ftpdatatask", test_ftpdatatask),
+            ("test_ftpdatataskDelegate", test_ftpdatataskDelegate),
+        ]
+    }
+    
+    let saveString =  "FTP implementation to test FTP upload,download and data tasks. Instead of sending a file, we are sending the hardcoded data.We are going to test FTP data,download and upload tasks with delegates & completion handlers. Creating the data here as we need to pass the count as part of the header.\r\n"
```

Comment:
Nit: extra space after `=`; inside the string, make sure you have one space after each comma and period.